graspy/cluster/autogmm.py

```diff
@@ -377,86 +373,6 @@ def __init__(
         self.verbose = verbose
         self.selection_criteria = selection_criteria
         self.max_agglom_size = max_agglom_size
-        self.n_jobs = n_jobs
-
-    def _fit_cluster(self, X, y, params):
```

Comment:
@caseypw can you explain? It looks like perhaps you were not on an up-to-date version of master before making all of your changes, can you revise to only reflect the changes you want to make?

Reply:
My bad, I'm not sure where these changes came from. Will get proper version of this file.